src/screens/ENSAssignRecordsSheet.js

```diff
@@ -89,63 +73,10 @@ export default function ENSAssignRecordsSheet() {
         style={{ paddingBottom: BottomActionHeight + 20 }}
       >
         <Stack space="19px">
-          <Pressable onPress={handleChooseCover}>
-            <Box
-              alignItems="center"
-              as={RadialGradient}
-              colors={[colors.whiteLabel + alpha, accentColor + alpha]}
-              height="126px"
-              justifyContent="center"
-              stops={[0.6, 0]}
-            >
-              <Text color="accent" size="18px" weight="heavy">
-                􀣵 Add Cover
-              </Text>
-            </Box>
-          </Pressable>
+          <CoverPhoto />
           <Bleed top={{ custom: 38 }}>
             <Box alignItems="center">
-              <Box
-                height={{ custom: avatarSize }}
-                width={{ custom: avatarSize }}
-              >
-                <Cover alignHorizontal="center">
-                  <BackgroundProvider color="body">
-                    {({ backgroundColor }) => (
-                      <Svg
-                        height="32"
-                        style={{ top: -6 }}
-                        viewBox="0 0 96 29"
-                        width="96"
-                      >
-                        <Path
-                          d="M9.22449 23.5H0V28.5H96V23.5H86.7755C85.0671 23.5 83.4978 22.5584 82.6939 21.051C67.8912 -6.70409 28.1088 -6.70408 13.3061 21.051C12.5022 22.5584 10.9329 23.5 9.22449 23.5Z"
-                          fill={backgroundColor}
-                        />
-                      </Svg>
-                    )}
-                  </BackgroundProvider>
-                </Cover>
-                <ButtonPressAnimation onPress={handleChooseAvatar}>
-                  <AccentColorProvider color={accentColor + '10'}>
-                    <Box
-                      alignItems="center"
-                      background="accent"
-                      borderRadius={avatarSize / 2}
-                      height={{ custom: avatarSize }}
-                      justifyContent="center"
-                      shadow="12px heavy accent"
-                      width={{ custom: avatarSize }}
-                    >
-                      <AccentColorProvider color={accentColor}>
-                        <Text color="accent" size="18px" weight="heavy">
-                          {` 􀣵 `}
-                        </Text>
-                      </AccentColorProvider>
-                    </Box>
-                  </AccentColorProvider>
-                </ButtonPressAnimation>
-              </Box>
+              <Avatar onChangeAvatarUrl={setAvatarUrl} />
```

Comment:
the avatar is not being set in state right? couldn't find the records being updated here maybe I'm missing something, same for the cover